scanner/src/lib/dataproducts/Battery_HealthData_v0.1.svelte

```diff
@@ -111,41 +117,14 @@
     color: white;
     font-style: normal;
 
-    .title {
-      font-size: 1rem;
-      font-weight: 500;
-      line-height: 1.5rem;
-      margin-bottom: 1rem;
-
-      &.no-bottom-margin {
-        margin-bottom: 0;
-      }
-    }
-
-    .subtitle {
-      font-size: 0.75rem;
-      font-weight: 400;
-      color: white;
-      margin-bottom: 1rem;
-
-      &.no-bottom-margin {
-        margin-bottom: 0;
-      }
-    }
-
-    .divider {
-      border-bottom: 1px solid #20303e;
-      margin-bottom: 1rem;
-    }
-
     .grid {
       display: grid;
       grid-template-columns: 1fr 1fr 1fr;
       gap: 1rem;
       margin-bottom: 1rem;
-      /* @media screen and (max-width: 450px) {
+      @media screen and (max-width: 330px) {
```

Comment:
330px is the min we support, it shouldn't be the `max-width` of a breakpoint. Please check again with Alina, it should be probably `max-width: 389px` . Alignment should only affect grid titles